docs/users-documentation.html

```diff
@@ -145,16 +133,16 @@
             <li>
             <p>
                 The easiest way to make the CLI tools running is to
-                <a href="https://github.com/project-ncl/dependency-analysis/raw/master/cli-wrap/da-cli-tools-0.8.1-pack.zip">download the ZIP archive</a>
-                from GitHub, which contains all the required support scripts and both CLI tools. The
-                latest versions of the tools are always placed in the folder /cli. When you download the
+                <a href="https://github.com/project-ncl/dependency-analysis/raw/master/cli-wrap/da-cli-tools-1.0.0-pack.zip">download the ZIP archive</a>
```

Comment:
The url shoul probably point to (not yet existing) tag 1.0.0. Or the 1.0.x branch